html final/styles.css

```
@@ -0,0 +1,853 @@
+@import url('https://fonts.google.com/specimen/Varela+Round#standard-styles');
+
+*{
+    font-family:Varela Round;
+
+}
+
+html{
+    scroll-behavior: smooth;
+
+}
+
+body{
+   width:100%;
+   height:1000px;
+   margin: 0px
```

with 0-s we can emit the px. Do not forget the ;
```suggestion
   margin: 0;
```